docker-compose.yml

```diff
@@ -57,20 +57,26 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:3e478a9
+    image: nebraltd/hm-diag:625ea72
+    depends_on:
+      - dbus-session
     environment:
-      - FIRMWARE_VERSION=2021.11.26.1
-      - DIAGNOSTICS_VERSION=3e478a9
+      - FIRMWARE_VERSION=2021.11.26.1-1
+      - DIAGNOSTICS_VERSION=625ea72
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
     volumes:
       - pktfwdr:/var/pktfwd
       - miner-storage:/var/data
+      - dbus:/session/dbus
```

Comment:
Don't think we need to expose the session bus, just the system bus for what this is used for

Reply:
The only thing using the session bus is the miner FYI